tools/parity_scale_decoder/src/generate_map.rs

```diff
@@ -148,6 +148,8 @@ pub fn generate_map() -> DumpDecodedMap {
         FindAllAssets,
         FindAllAssetsDefinitions,
         FindAllBlocks,
+        FindAllBlockHeaders,
+        FindBlockHeaderByHash,
```

Comment:
I'd like to keep this list in alphabetical order as it was before

Reply:
Right, didn't notice it was sorted